pages/types.ts

```diff
@@ -0,0 +1,42 @@
+import { StackableRedeemableObject, StackableRedeemableParams } from "@voucherify/sdk";
+
+export type EachProduct = {
+  productName: string;
+  productDescription: string;
+  quantity: number;
+  price: number;
+  src: string;
+  id: number;
+};
+
+export type Products = {
```

Comment:
Why Products is an object with one attribute? Overcomplicated IMHO

```ts
type Products = Product[]
```

Reply:
when i try to do like you i have ts error "property products does not exist on type "Products"" idk why but when it's like type Products = { products: Product[] } then it's okay